src/App.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import 'bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+import SetupPage from './SetupPage.js';
+import DeployPage from './DeployPage.js';
+import './App.css';
+
+export default class App extends React.Component {
+  render() {
+    let content = <SetupPage />;
+
+    if (window.location.search) {
+      let url = new URL(window.location);
+      const names = url.searchParams.getAll('name[]');
+      const urls = url.searchParams.getAll('url[]');
+      const deployments = names.map((name, idx) => ({ name, url: urls[idx] }));
+
+      content = (
+        <DeployPage
+          owner={url.searchParams.get('owner')}
+          repo={url.searchParams.get('repo')}
+          deployments={deployments}
+        />
+      );
+    }
```

Comment:
We need a solution to cope with loading shortlinks. When you go to `http://localhost:3000/s-xXq` it *used to* redirect to `http://localhost:3000/?name%5B%5D=Dev&name%5B%5D=Stage&name%5B%5D=Prod&owner=mozilla-services&url%5B%5D=https%3A%2F%2Fsymbols.dev.mozaws.net%2F__version__&url%5B%5D=https%3A%2F%2Fsymbols.stage.mozaws.net%2F__version__&url%5B%5D=https%3A%2F%2Fsymbols.prod.mozaws.net%2F__version__&repo=tecken`

I'm started to doubt the redirect, but we definitely need to make this page work. What it ought too do is load a component that wraps the `DeployPage` component, but upon mount it should do an XHR to get all the info that that shortlink represents. Perhaps, with that in place, we can just stay on `http://localhost:3000/s-xXq` without a redirect.

Reply:
I think the redirect was there because it was bolted on as an after-thought and it'd just basically help you fill in the fields in the setup form. 
Yeah, the more I think about it, let's not do a redirect. 

But, if you're on `http://localhost:3000/s-xXq` and click the header ("What's Deployed") perhaps it can go to `http://localhost:3000/` so the user can get back to the true home page.